.ci/jenkins/codeready_workspaces_seed_job.groovy

```diff
@@ -0,0 +1,182 @@
+String seedJobName = 'osio_ci_seed_job'
```

Comment:
I would consider renaming that file - it has nothing to do with codeready workspaces.

Reply:
done.